From 3a0457c02823a9467486089da69a7b15fcd441d7 Mon Sep 17 00:00:00 2001
From: Tom Moore <tmoore@locusrobotics.com>
Date: Wed, 27 Jul 2022 13:04:03 +0100
Subject: [PATCH] Fix conflicts

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 276329d..8eb7c88 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -16,7 +16,7 @@ export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
-export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG -O3
 ifneq ($(filter nocheck,$(DEB_BUILD_OPTIONS)),)
 	BUILD_TESTING_ARG=-DBUILD_TESTING=OFF -DCATKIN_ENABLE_TESTING=OFF
 endif
-- 
2.25.1

