From b81f25545d1b9a286ff40212078062e8da4f4814 Mon Sep 17 00:00:00 2001
From: Tom Moore <tmoore@locusrobotics.com>
Date: Thu, 15 Jun 2017 07:45:31 +0100
Subject: [PATCH] Update rules.em

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 991c991..02edf45 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -17,7 +17,7 @@ export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
-export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG -O3
 
 %:
 	dh  $@@
-- 
1.9.1

