From d147218058436eb27a712c3d87cf803ce60513d1 Mon Sep 17 00:00:00 2001
From: Tom Moore <tmoore@locusrobotics.com>
Date: Thu, 11 Aug 2016 11:45:28 +0100
Subject: [PATCH] Update rules.em

Enabling O3
---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 991c991..02edf45 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -17,7 +17,7 @@ export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
-export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG -O3
 
 %:
 	dh  $@@
-- 
2.7.4

